src/app/mork/api/v1/routers/users.py

```diff
@@ -0,0 +1,93 @@
+"""API routes related to users."""
+
+import logging
+from typing import Annotated
+from uuid import UUID
+
+from fastapi import APIRouter, Body, Depends, Path, Query
+from sqlalchemy import select, update
+from sqlalchemy.orm import Session
+
+from mork.auth import authenticate_api_key
+from mork.db import get_session
+from mork.models.users import (
+    ServiceName,
+    ServiceStatus,
+    User,
+)
+from mork.schemas.users import DeletionStatus, UserRead, UserUpdate
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(prefix="/users", dependencies=[Depends(authenticate_api_key)])
+
+
+@router.get("")
+@router.get("/")
+async def read_users(
```

Comment:
IMO adding a  `read_user` allowing to fetch a specific user can be useful. Doing this, we can have the state of a specific user at a specific time. 
We can also test at the beginning of each deletion process in each application if the user is still to process or not

Reply:
I can add a `read_user` as it sure will be useful.
Not sure about your second remark but we'll see that when implementing deletion on each app.